controllers/grid/pubIds/PubIdExportRepresentationsListGridCellProvider.inc.php

```diff
@@ -54,12 +54,12 @@ function getCellActions($request, $row, $column, $position = GRID_ACTION_POSITIO
 				if (empty($title)) $title = __('common.untitled');
 				$authorsInTitle = $publishedSubmission->getShortAuthorString();
 				$title = $authorsInTitle . '; ' . $title;
-				import('lib.pkp.controllers.grid.submissions.SubmissionsListGridCellProvider');
+				import('lib.pkp.classes.core.ServicesContainer');
 				return array(
 					new LinkAction(
 						'itemWorkflow',
 						new RedirectAction(
-							SubmissionsListGridCellProvider::getUrlByUserRoles($request, $publishedSubmission)
+							ServicesContainer::instance()->getWorklowUrlByUserRoles($publishedSubmission)
```

Comment:
Woops! I've done this before. Just missed the call to get the submission service. :)